proto/squad/liquidity/v1beta1/tx.proto

```diff
@@ -66,6 +69,29 @@ message MsgCreatePool {
 // MsgCreatePoolResponse defines the Msg/CreatePool response type.
 message MsgCreatePoolResponse {}
 
+// MsgCreateRangedPool defines an SDK message for creating a ranged pool.
+message MsgCreateRangedPool {
+  // creator specifies the bech32-encoded address that is the pool creator
+  string creator = 1;
+
+  // pair_id specifies the pair id.
+  uint64 pair_id = 2;
+
+  // deposit_coins specifies the amount of coins to deposit.
+  repeated cosmos.base.v1beta1.Coin deposit_coins = 3
+      [(gogoproto.castrepeated) = "github.com/cosmos/cosmos-sdk/types.Coins", (gogoproto.nullable) = false];
+
+  string initial_price = 4
+      [(gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Dec", (gogoproto.nullable) = false];
+
+  string min_price = 5 [(gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Dec"];
+
+  string max_price = 6 [(gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Dec"];
```

Comment:
Leaving some comments to provide information for these values for a pool creator will be helpful. Is it important to note that the initial price doesn't necessarily need to be in a range of `min_price` and `max_price`?

Reply:
I think that's not that important, since `initialPrice` is automatically adjusted to `minPrice` or `maxPrice` when it's out of the range. And maybe later, the option to specify `initialPrice` which is out of the range could be removed.

Reply:
I give this suggestion since `initialPrice` is required value in `NewCreateRangedPoolCmd` CLI command.